src/Certify.Models/Models/Config/ChallengeProviders.cs

```diff
@@ -55,12 +57,29 @@ public class ChallengeProviders
                 HelpUrl="https://api.cloudflare.com/#dns-records-for-a-zone-dns-record-details",
                 ProviderParameters = new List<ProviderParameter>{
                     new ProviderParameter{Key="emailaddress", Name="Email Address", IsRequired=true },
-                    new ProviderParameter{Key="authkey", Name="Auth Key", IsRequired=true }
+                    new ProviderParameter{Key="authkey", Name="Auth Key", IsRequired=true },
+                    new ProviderParameter{Key="propdelay", Name="Propegation Delay", IsRequired=true, Value="60" }
```

Comment:
We will make override of propagation delay available seperately as advanced UI, there are other situations where an adjustable wait time between steps will be required (even for non-DNS).